network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -1,19 +1,42 @@
 
 from django.db import models
 
-from wagtail.core.models import Page
-from wagtail.admin.edit_handlers import FieldPanel
-from wagtail.images.edit_handlers import ImageChooserPanel
-from wagtail.documents.edit_handlers import DocumentChooserPanel
-from wagtail.core.fields import RichTextField
+from modelcluster.fields import ParentalKey
 
+from wagtail.admin.edit_handlers import FieldPanel, InlinePanel, MultiFieldPanel
+from wagtail.core.fields import RichTextField
+from wagtail.core.models import Orderable, Page
+from wagtail.documents.edit_handlers import DocumentChooserPanel
+from wagtail.images.edit_handlers import ImageChooserPanel
+from wagtail.snippets.edit_handlers import SnippetChooserPanel
 
 from ..mixin.foundation_metadata import FoundationMetadataPageMixin
+from networkapi.wagtailpages.models import BlogAuthor
+from networkapi.wagtailpages.pagemodels.publications.article import ArticlePage
+
+
+class PublicationAuthors(Orderable):
+    """This allows us to select one or more blog authors from Snippets."""
+
+    page = ParentalKey("wagtailpages.PublicationPage", related_name="authors")
+    author = models.ForeignKey(
+        BlogAuthor,
+        on_delete=models.SET_NULL,
+        null=True,
+        blank=False
```

Comment:
Authors should be optional, according to specs not every publication is going to have specific authors cited